webapp/src/components/expanded_view/controls_button.tsx

```diff
@@ -22,8 +22,23 @@ export type Props = {
 }
 
 export default function ControlsButton(props: Props) {
+    const overlayRef = useRef();
+
+    const onClick = () => {
+        if (props.disabled) {
+            return;
+        }
+
+        // @ts-ignore
+        overlayRef.current?.hide();
+        props.onToggle?.();
+    };
```

Comment:
Could you explain to a mere mortal what's happening here? :p

Reply:
Heh -- it's so that once we click on the control button, the tooltip overlay will go away. Before, when you clicked the reactions button, the reaction bar would appear but the tooltip was still there. I have to put ts-ignore because they (react-bootstrap) doesn't publicly expose what we're doing here, it's kind of internal.

Reply:
Interesting, I wonder if a similar approach could perhaps fix https://mattermost.atlassian.net/browse/MM-48336 :thinking:

Reply:
Yep, probably. I'll take a look at that. 👍